companion/src/cmake/Macros.cmake

```diff
@@ -24,3 +24,15 @@ MACRO (NOW RESULT)
     ENDIF (WIN32)
 ENDMACRO (NOW)
 
+MACRO (GIT_ID RESULT)
+    IF (WIN32)
+        EXECUTE_PROCESS(COMMAND "cmd" " /C git rev-parse HEAD" OUTPUT_VARIABLE ${RESULT})
```

Comment:
Yes I know. I copied the code from somewhere else to avoid duplicate, but didn't change the style. This will be done later, don't worry